ios/Classes/SwiftFlutterVideoCompressPlugin.swift

```diff
@@ -302,6 +303,13 @@ public class SwiftFlutterVideoCompressPlugin: NSObject, FlutterPlugin {
         
         let destinationPath = trimmedRegift.createGif();
         
+        // remove compressed video since it's not anymore needed.
```

Comment:
This file is generated by the image_picker lib. I think the original video should not be deleted if the plugin not used.

Reply:
makes sense. I'll remove it.